Sources/VLCMovieViewController.m

```diff
@@ -1121,11 +1150,9 @@ - (void)volumeSliderAction:(id)sender
 
 - (void)updateTimeDisplayButton
 {
+    NSString *elapsedTime = _displayRemainingTime ? [[_vpc remainingTime] stringValue] : [[_vpc playedTime] stringValue];
     UIButton *timeDisplayButton = self.timeNavigationTitleView.timeDisplayButton;
-    if (_displayRemainingTime)
-        [timeDisplayButton setTitle:[[_vpc remainingTime] stringValue] forState:UIControlStateNormal];
-    else
-        [timeDisplayButton setTitle:[[_vpc playedTime] stringValue] forState:UIControlStateNormal];
+    [timeDisplayButton setTitle:elapsedTime forState:UIControlStateNormal];
```

Comment:
I would put that in a separate commit since it has nothing to do with this PR/ commit message